dbt_project.yml

```diff
@@ -1,17 +1,21 @@
 name: 'dbt_date'
 version: '0.1'
 
+config-version: 2
+
 target-path: "target"
 clean-targets: ["target", "dbt_modules"]
 macro-paths: ["macros"]
 log-path: "logs"
 
 require-dbt-version: ">=0.14.0"
 profile: dev
+
 quoting:
     identifier: false
     schema: false
 
-models:
-    vars:
-        'dbt_date:time_zone': 'America/Los_Angeles'
\ No newline at end of file
+vars:
```

Comment:
what does this change exactly?

Reply:
- updating to config format per dbt doc: https://docs.getdbt.com/docs/guides/migration-guide/upgrading-to-0-17-0
- there is a deprecation warning for using config format version 1

Reply:
Specific to `vars`, config format version 2 avoids ambiguous scoping issues as stated in the doc